### PR TITLE
SEO-2336 Blazor Tooltip External Linking

### PR DESCRIPTION
**Pull request by Scholar-jaruha**
Hi @ChristoIssac ,
Please review changes made on blazor tooltip GitHub.
Regards,
@Scholar-jaruha

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -20,3 +14,52 @@ To run this application, you need to first clone or download the ‘Create a Too
...
...tm_source=github&utm_medium=listing&utm_campaign=blazor-popups-github), and [MAUI(Preview)](https://www.syncfusion.com/maui-controls?utm_source=github&utm_medium=listing&utm_campaign=blazor-split-buttons-github)), and desktop development ([WinForms](https://www.syncfusion.com/winforms-ui-controls?utm_source=github&utm_medium=listing&utm_campaign=blazor-popups-github), [WPF](https://www.syncfusion.com/wpf-controls?utm_source=github&utm_medium=listing&utm_campaign=blazor-popups-github), [WinUI](https://www.syncfusion.com/winui-controls?utm_source=github&utm_medium=listing&utm_campaign=blazor-popups-github), [Flutter](https://www.syncfusion.com/flutter-widgets?utm_source=github&utm_medium=listing&utm_campaign=blazor-popups-github) and [UWP](https://www.syncfusion.com/uwp-ui-controls?utm_source=github&utm_medium=listing&utm_campaign=blazor-popups-github), and [MAUI(Preview)](https://www.syncfusion.com/maui-controls?utm_source=github&utm_medium=listing&utm_campaign=blazor-split-buttons-github)). We provide ready-to-deploy enterprise software for dashboards, reports, data integration, and big data processing. Many customers have saved millions in licensing fees by deploying our software.
```
Comment (Girija-Rajendran) [inline README.md]: invalid utm_campaign

File: README.md [hunk 2]
```diff
@@ -20,3 +14,52 @@ To run this application, you need to first clone or download the ‘Create a Too
...
+
+[View Demos](https://github.com/SyncfusionExamples/Create-a-Tooltip-Component-in-a-Blazor-Server-Application?utm_source=github&utm_medium=listing&utm_campaign=blazor-tooltip-github-samples)
+
+[Community Forums](https://www.syncfusion.com/forums/blazor-components?utm_source=github&utm_medium=listing&utm_campaign=blazor-tooltip-github-samples)
+
+[Suggest a feature or report a bug](https://www.syncfusion.com/feedback/blazor-components?utm_source=github&utm_medium=listing&utm_campaign=blazor-tooltip-github-samples)
+
+[Online example](https://blazor.syncfusion.com/demos/toolbar/default-functionalities?utm_source=github&utm_medium=listing&utm_campaign=blazor-tooltip-github-samples)
+
+[Video Tutorials](https://www.syncfusion.com/tutorial-videos/blazor/toolbar?utm_source=github&utm_medium=listing&utm_campaign=blazor-tooltip-github-samples)
```
Comment (Girija-Rajendran) [inline README.md]: check the demo and video links 
image

File: README.md (hunk 1 above)
Comment (Scholar-jaruha) [reply README.md]: UTM campaign changed

File: README.md (hunk 2 above)
Comment (Scholar-jaruha) [reply README.md]: control name changed

File: README.md [hunk 3]
```diff
@@ -20,3 +14,56 @@ To run this application, you need to first clone or download the ‘Create a Too
...
...tip-github-samples), and [.NET MAUI(Preview)](https://www.syncfusion.com/maui-controls?utm_source=github&utm_medium=listing&utm_campaign=blazor-tooltip-github)), and desktop development ([WinForms](https://www.syncfusion.com/winforms-ui-controls?utm_source=github&utm_medium=listing&utm_campaign=blazor-tooltip-github-samples), [WPF](https://www.syncfusion.com/wpf-controls?utm_source=github&utm_medium=listing&utm_campaign=blazor-tooltip-github-samples), [WinUI](https://www.syncfusion.com/winui-controls?utm_source=github&utm_medium=listing&utm_campaign=blazor-tooltip-github-samples), [Flutter](https://www.syncfusion.com/flutter-widgets?utm_source=github&utm_medium=listing&utm_campaign=blazor-tooltip-github-samples) and [UWP](https://www.syncfusion.com/uwp-ui-controls?utm_source=github&utm_medium=listing&utm_campaign=blazor-tooltip-github-samples), and [.NET MAUI(Preview)](https://www.syncfusion.com/maui-controls?utm_source=github&utm_medium=listing&utm_campaign=blazor-tooltip-github-samples)). We provide ready-to-deploy enterprise software for dashboards, reports, data integration, and big data processing. Many customers have saved millions in licensing fees by deploying our software.
```
Comment (Girija-Rajendran) [inline README.md]: check this line 
image

File: README.md [hunk 4]
```diff
@@ -20,3 +14,56 @@ To run this application, you need to first clone or download the ‘Create a Too
...
+[Suggest a feature or report a bug](https://www.syncfusion.com/feedback/blazor-components?utm_source=github&utm_medium=listing&utm_campaign=blazor-tooltip-github-samples)
+
+[Online example](https://blazor.syncfusion.com/demos/tooltip/default-functionalities?utm_source=github&utm_medium=listing&utm_campaign=blazor-tooltip-github-samples)
+
+[Video Tutorials](https://www.syncfusion.com/tutorial-videos/blazor/tooltip?utm_source=github&utm_medium=listing&utm_campaign=blazor-tooltip-github-samples)
+
+[Knowledge base](https://www.syncfusion.com/kb/blazor-components?utm_source=github&utm_medium=listing&utm_campaign=blazor-tooltip-github-samples)
+
+### About Syncfusion Blazor Components
+The Syncfusion's [Blazor components](https://www.syncfusion.com/blazor-components?utm_source=github&utm_medium=listing&utm_campaign=blazor-tooltip-github-samples) library offers over 70 UI components to work with Blazor server-side and client-side (Blazor WebAssembly) projects seamlessly. In addition to Tooltip, we provide popular Blazor Components such as DataGrid, Chart, Scheduler, Diagram and Word Processor.
```
Comment (Girija-Rajendran) [inline README.md]: FT link?
image

**Review by Girija-Rajendran — APPROVED** [review]

**Pull request merged**